### DISCUSSION TITLE
mqtt integration

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#14309] [Feature Requests]
I would like to use mqtt on the client to respond to events in real time, with the same token-based authorization system

**Comment by rijkvanzanten** [comment]
I've put a note on the bigger discussion on real time updates 🙂 https://github.com/directus/directus/discussions/3221

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
